pylhc/plot_spectrum.py

```diff
@@ -110,7 +107,6 @@ def plot_line(ax, label, resonance, tunes):
 def save_plot(fname):
     plt.tight_layout()
     plt.savefig(fname)
-    plt.show()
     plt.close()
```

Comment:
There should be no plots to close now ..?

Btw. wouldn't it make more sense to give this function the figure itself and call `tight_layout()` and `savefig()` on it. I am not a big fan of the `plt-thing. But personal preference.